### PR TITLE
Fix deprecated swiftmailer parameter

### PR DESCRIPTION
**Pull request by piotrantosik**
fixed pr https://github.com/symfony/symfony-standard/pull/1037 @xabbuh please review

### CODE REVIEW

**Review by xabbuh — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @piotrantosik.

File: composer.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
         "doctrine/orm": "^2.4.8",
         "doctrine/doctrine-bundle": "~1.4",
         "symfony/assetic-bundle": "~2.3",
-        "symfony/swiftmailer-bundle": "~2.3",
+        "symfony/swiftmailer-bundle": "~2.3,>=2.3.10",
```
Comment (bocharsky-bw) [inline composer.json]: Why not just `~2.3.10`?

Comment (xabbuh) [inline composer.json]: `~2.3.10` would forbid to install `2.4.0`, for example, once it got released.

Comment (bocharsky-bw) [inline composer.json]: @xabbuh Ah, it makes sense. But `^2.3.10` would allow `2.4.0`, right?

Comment (xabbuh) [inline composer.json]: Yes, that would be equivalent.

Comment (bocharsky-bw) [inline composer.json]: @xabbuh Thanks and happy New Year! :)

Comment (xabbuh) [inline composer.json]: @bocharsky-bw Happy new year for you too! :)

Comment (Pierstoval) [inline composer.json]: Yep I also recommend `^2.3.10` which is nicer :)

Comment (fabpot) [inline composer.json]: done in more recent versions of SE

**Pull request merged**